src/lib/pickles/unix/graphql_scalars.mli

```diff
@@ -0,0 +1,10 @@
+(* Undocumented *)
```

Comment:
I think this is more noise than signal. Delete.

Reply:
Done